### DISCUSSION TITLE
What version of cppcheck is being used?

### DISCUSSION THREAD

**Discussion by chmorgan** [linuxdeepin/developer-center#3401] [Q&A | 问答板块]
Wanted to see what version of cppcheck was being used.

**Comment by BLumia** [comment]
According to the [action configuration](https://github.com/linuxdeepin/action-cppcheck/blob/4a81a73e6bc3eaa2176af71598be76259b55b957/action.yml#L38-L42) file and [one of the CI log](https://github.com/linuxdeepin/dde-file-manager/runs/7912236836?check_suite_focus=true#step:4:50) that using the cppcheck action, currently `cppcheck_1.90-4build1_amd64.deb` from ubuntu focal (20.04) was being used.

btw it's a question instead of feature request so I'll move this thread to the related category :)

**Reply by chmorgan** [reply]
@BLumia is it possible to use a newer version of cppcheck?

**Reply by BLumia** [reply]
cc @myml

**Reply by myml** [reply]
> @BLumia is it possible to use a newer version of cppcheck?

Maybe we can wait for github to switch the default image to ubuntu 22.04? It won't be long. https://github.com/actions/runner-images/issues/5998#issuecomment-1231939134
